lang/ilias_pt.lang

```diff
@@ -1960,7 +1960,6 @@ auth#:#login_page_switch_rte#:#Se o servidor de chat estiver acessível apenas a
 auth#:#lti_consumer_inactive#:#O consumidor da ferramenta LTI está desativado.
 auth#:#saml_tab_head_idp#:#IDP
 awrn#:#awareness_now_online#:#Online agora
-awrn#:#awareness_now_online_users#:#[ONLINE_USER_NAMES]
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.